File: Allamvizsga/Allamvizsga/Models/ServicesModels.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace Allamvizsga.Models
{
    public class ServicesModels
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }

        public string Identifier { get; set; }
        public string VIN { get; set; }

       

        public DateTime ServiceDate { get; set; }

        public int Price { get; set; }

        public int ActualKm { get; set; }

        public int Flag { get; set; }

        public int ServiceId { get; set; }

        public string PhoneNumber { get; set; }
        [ForeignKey("PhoneNumber")]
        public virtual OwnerModels Owner { get; set; }


    }
}

File: Allamvizsga/Allamvizsga/Scripts/Services/ServicesModels.js
function ServicesModels(data) {
    this.id = ko.observable(null);
    this.vin= ko.observable(null);
    this.owner=null;
    this.identifier = ko.observable(null);
    this.serviceDate = ko.observable(null);
    this.price = ko.observable(null);
    this.actualKm = ko.observable(null);
    this.flag = ko.observable(null);
    this.serviceId = ko.observable(null);
   if (data != null) {

        this.id(data.ID)
        this.vin(data.VIN);
        this.owner=new OwnerModel(data.Owner);
        this.identifier(data.Identifier);
        this.serviceDate(data.ServiceDate);
        this.price(data.Price);
        this.actualKm(data.ActualKm);
        this.flag(data.Flag);
        this.serviceId(data.ServiceId);
    }
    
 }